CDP4Orm/AutoGenDao/IDataModelUtils.cs

```diff
@@ -35,8 +56,8 @@ public interface IDataModelUtils
         /// The partition name for the passed in concrete type, otherwise null
         /// </returns>
         /// <remarks>
-        /// Null is returned if there is no type map entry or there are multiple
+        /// A collection of possible partitions as string
         /// </remarks>
-        string GetSourcePartition(string typeName);
+        IList<string> GetSourcePartition(string typeName);
```

Comment:
my preference would be to return an IEnumerable instead of IList, if you agree please change, if not, leave it as is and I go with your opinion.

Reply:
Well, IList is indeed "wrong", but I started using IReadOnlyList in these cases as ReSharper reminds us on every case where we use IEnumerable as a return type that it might read the data multiple times. In case of Linq statements this means that data might be retrieved from the database multiple times. I tend to make sure ReSharper does not complain about this possible scenario anymore, so an IReadOnlyList is the best option imho.